src/connectedmachine/azext_connectedmachine/generated/_help.py

```diff
@@ -0,0 +1,127 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=too-many-lines
+
+from knack.help_files import helps
+
+
+helps['connectedmachine machine'] = """
+    type: group
+    short-summary: connectedmachine machine
+"""
+
+helps['connectedmachine machine list'] = """
+    type: command
+    short-summary: "Lists all the hybrid machines in the specified subscription. Use the nextLink property in the \
+response to get the next page of hybrid machines."
+    examples:
+      - name: List Machines by resource group
+        text: |-
+               az connectedmachine machine list --resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine show'] = """
+    type: command
+    short-summary: "Retrieves information about the model view or the instance view of a hybrid machine."
+    examples:
+      - name: Get Machine
+        text: |-
+               az connectedmachine machine show --name "myMachine" --resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine delete'] = """
+    type: command
+    short-summary: "The operation to remove a hybrid machine identity in Azure."
+    examples:
+      - name: Delete a Machine
+        text: |-
+               az connectedmachine machine delete --name "myMachine" --resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine-extension'] = """
+    type: group
+    short-summary: connectedmachine machine-extension
+"""
+
+helps['connectedmachine machine-extension list'] = """
+    type: command
+    short-summary: "The operation to get all extensions of a non-Azure machine"
+    examples:
+      - name: GET all Machine Extensions
+        text: |-
+               az connectedmachine machine-extension list --machine-name "myMachine" --resource-group \
+"myResourceGroup"
+"""
+
+helps['connectedmachine machine-extension show'] = """
+    type: command
+    short-summary: "The operation to get the extension."
+    examples:
+      - name: GET Machine Extension
+        text: |-
+               az connectedmachine machine-extension show --machine-name "myMachine" --name "CustomScriptExtension" \
+--resource-group "myResourceGroup"
+"""
+
+helps['connectedmachine machine-extension create'] = """
+    type: command
+    short-summary: "The operation to create or update the extension."
+    examples:
+      - name: Create or Update a Machine Extension
```

Comment:
I think we should remove `or update` here. @qiaozha can we do this in codegen automatically?

Reply:
Make sense. I filed an issue to track this https://github.com/Azure/autorest.az/issues/553

Reply:
Suddenly realized the example name is from swagger example reference name. https://github.com/Azure/azure-rest-api-specs/blob/master/specification/hybridcompute/resource-manager/Microsoft.HybridCompute/stable/2020-08-02/HybridCompute.json#L90. Not sure we can change it as we use this to identify the example. 
If you mean the short summary I think we can change it.

Reply:
If you check the example in the update command below, it's using the same name. It's better to distinguish them. Please evaluate further the feasibility of change as this is a common case in swagger.

Reply:
Let me know if there's something I need to do here